### PR TITLE
GitHub#51112: corrected incorrect string syntax

### PR DESCRIPTION
**Pull request by ahardin-rh**
Addresses https://github.com/openshift/openshift-docs/issues/51112

Version(s): 4.11+

Link to docs preview: https://51145--docspreview.netlify.app/openshift-enterprise/latest/web_console/dynamic-plug-ins.html#dynamic-plug-in-sdk-extensions_dynamic-plugins

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@bmcelvee FYI on the syntax for the escaped pipe, which fixes the associated issue: https://docs.asciidoctor.org/asciidoc/latest/tables/data-format/#escape-the-cell-separator

File: modules/dynamic-plug-in-sdk-extensions.adoc [hunk 1]
```diff
@@ -137,13 +139,13 @@ which returns actions for the given resource model
...
 |Name |Value Type |Optional |Description
-|`catalogId` |`string | string[]` |no |The unique identifier for the
+|`catalogId` |`string` \| `string[]` |no |The unique identifier for the
 catalog this provider contributes to.
 
 |`type` |`string` |no |Type ID for the catalog item type.
 
 |`filter` |`CodeRef<(item: CatalogItem) => boolean>` |no |Filters items
-of a specific type. Value is a function that takes CatalogItem[] and
+of a specific type. Value is a function that takes `CatalogItem[]`` and
```
Comment (bmcelvee) [inline modules/dynamic-plug-in-sdk-extensions.adoc:148]: Sneaky extra backtick (`)

**Comment by bmcelvee** [comment]
Just one sneaky extra backtick, and I'm not sure if the build had caught up yet. Otherwise, LGTM!

**Comment by ahardin-rh** [comment]
No QE or SME review required, as this is a typo fix

**Pull request merged**